start-site/src/main/resources/application.yml

```diff
@@ -436,6 +436,14 @@ initializr:
               description: Authenticating a User with LDAP
             - rel: reference
               href: https://docs.spring.io/spring-boot/docs/{bootVersion}/reference/htmlsingle/#web.security
+        - name: Spring Authorization Server
+          id: oauth2-authorization-server
+          description: Spring Boot integration for Spring Security's authorization server features.
+          facets:
+            - native
+          links:
```

Comment:
Support was added in Boot `3.1.0-M2` but really only worked since `3.1.0-RC1`. Should there be a `compatibilityRange` added?

Reply:
> Support was added in Boot 3.1.0-M2 but really only worked since 3.1.0-RC1. 

@sjohnr I had concerns about adding `compatibilityRange`. Could you please let me know if there is a good way to add this?

Reply:
You can use `compatibilityRange: "3.1.0-RC1"` which means `3.1.0-RC1` and later.